lib/src/main/java/com/futurewei/alcor/common/db/ignite/IgniteConfiguration.java

```diff
@@ -109,19 +136,17 @@ private IgniteClient getThinIgniteClient() {
                     .setSslTrustCertificateKeyStorePassword(trustStorePassword);
         }
 
-        IgniteClient igniteClient = null;
-
-        try {
-            igniteClient = Ignition.startClient(cfg);
-        } catch (ClientException e) {
-            logger.log(Level.WARNING, "Start client failed:" + e.getMessage());
-        } catch (Exception e) {
-            logger.log(Level.WARNING, "Unexpected failure:" + e.getMessage());
-        }
+        return Ignition.startClient(cfg);
+    }
 
-        Assert.notNull(igniteClient, "IgniteClient is null");
+    private IgniteClient getKubernetesThinIgniteClient() {
+        KubernetesConnectionConfiguration kcfg = new KubernetesConnectionConfiguration();
+        kcfg.setNamespace(kubeNamespace);
+        kcfg.setServiceName(kubeServiceName);
 
-        return igniteClient;
+        ClientConfiguration ccfg = new ClientConfiguration();
+        ccfg.setAddressesFinder(new ThinClientKubernetesAddressFinder(kcfg));
+        return Ignition.startClient(ccfg);
```

Comment:
If Ignite doesn't print the connection parameters by default, could we add log here to show the connection parameters?

Reply:
Will add a log.